tests/NgTests/NgTests.csproj

```diff
@@ -196,20 +196,17 @@
     <PackageReference Include="Lucene.Net.Contrib">
       <Version>3.0.3</Version>
     </PackageReference>
-    <PackageReference Include="Microsoft.ApplicationInsights">
```

Comment:
Transitive dependency of NuGet.Services.Logging. Upgraded to version 2.10.0 as a result.